apps/tangle-dapp/context/useNetworkStore.ts

```diff
@@ -0,0 +1,22 @@
+'use client';
+
+import { Network } from '@webb-tools/webb-ui-components/constants';
+import { create } from 'zustand';
```

Comment:
Do you think we need the `zustand` library for state management here? Previously, we utilized `rxjs` along with the [observable hook](https://observable-hooks.js.org/guide/) to manage the shared state. I believe it's crucial to maintain a single source of truth across dApps. What are your thoughts on this, @vutuanlinh2k2, @yurixander, @devpavan04?

Reply:
I think in the futre we can use `zustand ` in simple cases like this one for faster development. `rxjs` and `observable-hooks` can be used in more complex cases such as real-time data, etc.